client/app/components/projects/planning-project-card.hbs

```diff
@@ -0,0 +1,17 @@
+<li
+  class="large-margin-bottom"
+  data-test-project-name={{@project.name}}
+  data-test-type='working-on-it'
+>
+  <h5 class="no-margin">
+    {{@project.name}}
+    <small class="text-weight-normal">{{@project.id}}</small>
+  </h5>
+
+  <p class="tiny-margin-bottom text-small">
+    <span class="text-dark-gray">Primary Applicant:</span> 
+    <strong>{{@project.primaryApplicant}}</strong>
+  </p>
+
+  <a href="/pas.html" class="button hollow tiny">View Pre-Application Statement</a>
```

Comment:
This button will need to be visible based on the availability of packages in general

Reply:
From #17:
> Project shows up in the right "Working on it..." column with no button if
Project is missing PAS data because the PAS package isn't visible or hasn't been created yet

The api will give us packages as an array, and it's possible that there's packages that are not PAS packages -- so have to filer if there no packages where type = 'pas'

Reply:
Resolved in new branch